ql/src/test/org/apache/hadoop/hive/ql/exec/tez/TestTezTask.java

```diff
@@ -217,7 +217,7 @@ public void testBuildDag() throws IllegalArgumentException, IOException, Excepti
   }
 
   @Test
-  public void testEmptyWork() throws IllegalArgumentException, IOException, Exception {
+  public void testEmptyWork() throws IllegalArgumentException, Exception {
```

Comment:
Listing `IllegalArgumentException` is not necessary if `Exception` can be thrown.

Reply:
Fixed.